Automattic/jetpack
# Identity_Crisis: Initial Skeleton

Fixes #

#### Changes proposed in this Pull Request:

* Adds the Identity_Crisis package.

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






1) Add the following filters to your site via mu-plugin, theme functions, etc

add_filter( 'jetpack_sync_home_url', function ( $value ) {
		if( defined('XMLRPC_REQUEST') && XMLRPC_REQUEST ) {
			return $value;
		}
		return 'http://example.org/home';
	} );

	add_filter( 'jetpack_sync_site_url', function ( $value ) {
		if( defined('XMLRPC_REQUEST') && XMLRPC_REQUEST ) {
			return $value;
		}
		return 'http://example.org/site';
	} );
	
2) Perform a syncable action( edit post, etc )
3) Verify your site is put into safe mode.

#### Proposed changelog entry for your changes:


* Sync: Adding the Identity_Crisis package.
